content/en/docs/how-tos/cluster-claim.md

```diff
@@ -214,9 +214,35 @@ document.
 
 The cluster pools are maintained by Hive behind the scenes, so installation failures, cloud platform account
 misconfigurations and similar issues are not exposed to actual CI jobs: the jobs will simply never successfully claim a
-cluster if Hive fails to install them. Currently, cluster pool owners cannot access the logs or other troubleshooting
-information on the `hive` cluster. For now, work with DPTP (ping `@dptp-helpdesk` on #forum-testplatform Slack channel)
-to troubleshoot pools that never get ready clusters.
+cluster if Hive fails to install them.
+
+The `ClusterRole` `hive-cluster-pool-admin` will [contains the permissions](https://github.com/openshift/hive/blob/master/docs/clusterpools.md#managing-admins-for-cluster-pools) to all resources created for a pool.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: hive-cluster-pool-admin-0
+  namespace: dptp-demo-cluster-pools
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: hive-cluster-pool-admin
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: petr
+```
+
+The pool owners can list the namespace to filter out which namespaces are created for their pools.
+Login to [the `hive` cluster](https://console-openshift-console.apps.hive.9xw5.p1.openshiftapps.com/dashboards) and add the user into
+[the cluster-namespace-viewer group](https://github.com/openshift/release/tree/master/clusters/hive/pools/hypershift/hypershift-pool_admins.yaml).
```

Comment:
For the actual troubleshooting, let's assume that the user is already subject to the RBAC; if not, they need to be added first via gitops. Let's not link to an actual hypershift RBACs.

Reply:
I linked to the wrong file.
Will update.